### PR TITLE
Fix for insertion new item before the last item

### PR DESCRIPTION
**Pull request by alex-ionochkin**
@sinan, can you review this fix? It's for a case when new item should be inserted before the last item in the list. Currently new item is added to the end of the list in this case

### CODE REVIEW

File: src/components/list/listview.coffee [hunk 1]
```diff
@@ -104,9 +104,9 @@ module.exports = class KDListView extends KDView
     {boxed, lastToFirst} = @getOptions()
 
 
-    isLastIndex  = index >= @items.length - 1
+    isLastIndex  = index > @items.length - 1
```
Comment (usirin) [inline src/components/list/listview.coffee]: Can you move cache the result of `length - 1` and use that if possible, I know it was already like that but changing it to a local variable would be better for `DRY`.

**Comment by usirin** [comment]
Other than that minor stuff, it LGTM. Thanks for that @alex-ionochkin :+1:

**Comment by alex-ionochkin** [comment]
@usirin, thanks for review. I cached `length - 1` as you suggested. Please take a look

**Comment by usirin** [comment]
LGTM

**Pull request merged**